src/routerd_lib/handlers/proxy.cpp

```diff
@@ -170,8 +311,8 @@ namespace NAC {
         bool contentDispositionFormData
     ) const {
         ServiceReplied(request, serviceName);
-
-        if ((serviceName == std::string("output")) && !request->IsResponseSent()) { // TODO
+        if (serviceName == std::string("output") && !request->IsResponseSent()
+        ) { // TODO
```

Comment:
Weird formatting, it should be as it was: `if ((serviceName == std::string("output")) && !request->IsResponseSent()) {`